src/components/DashboardHome/Sidebar.jsx

```diff
@@ -5,7 +5,7 @@ import linksData from './sampleLinks';
 
 const Sidebar = () => (
   <>
-    <Links title={linksData.title} links={linksData.links} />
+    <Links id={linksData.id} title={linksData.title} links={linksData.links} />
```

Comment:
I am not sure why we need this required `id` prop?

Reply:
@muhammad-ammar For a11y, the `Links` component has an `aria-controls` attribute on the "show less / show all" button that needs to refer to an element with a unique id. I'm adding this `id` prop as way to pass in a unique value to the component if/when there are multiple `Links` components on the page.